examples/all-clusters-app/esp32/README.md

```diff
@@ -1,29 +1,30 @@
 # CHIP All Clusters Example
 
-A prototype application that uses CHIP to setup WiFi on the ESP32 and runs an
-Echo Server. This example will evolve as more complex messaging is supported in
-CHIP.
+A prototype application that demonstrates device commissioning and cluster
+control.
 
 ---
 
--   [CHIP App Server Example](#chip-app-server-example)
+-   [CHIP All Clusters Example](#chip-all-clusters-example)
     -   [Supported Devices](#supported-devices)
     -   [Building the Example Application](#building-the-example-application)
         -   [To build the application, follow these steps:](#to-build-the-application-follow-these-steps)
-    -   [Using the Echo Server](#using-the-echo-server)
-        -   [Connect the ESP32 to a 2.4GHz Network of your choice](#connect-the-esp32-to-a-24ghz-network-of-your-choice)
-        -   [Use the ESP32's Network](#use-the-esp32s-network)
+    -   [Commissioning and cluster control](#commissioning-and-cluster-control)
+        -   [Setting up python controller](#setting-up-python-controller)
```

Comment:
Can we remove `To build the application, follow these steps:` from being a separate section? I don't think that a separate heading is required for that

Reply:
Done